src/cmd_definitions.c

```diff
@@ -23,6 +24,11 @@ const cmd_definition *cmd_get_definition(char *cmd_name) {
   return NULL;
 }
 
+void cmd_help(char **arg_list, size_t arg_count, char *res_msg) {
+  sceAppMgrDestroyOtherApp();
+  strcpy(res_msg, "Command\t\tArguments\tExplanation\n\nhelp\t\tNone\t\tDisplay this help screen\ndestroy\t\tNone\t\tKill all running applications\nlaunch\t\t<TITLEID>\tLaunch an app by Title ID\nreboot\t\tNone\t\tReboot the console\nscreen\t\ton or off\tTurn screen on or off\n");
```

Comment:
I'm not fond of this hardcoding because the same information is now in 3 different places:
– README.md.
– `const cmd_definition cmd_definitions[]`.
– And now here.

Reply:
And then fix this to read from `cmd_definitions`.

Reply:
I don't think I can take care of that.  Might be wise to just drop that.   Ive mostly moved on to the switch.  If you want to take care of this thats fine. But. Sorry :/